model/user.go

```diff
@@ -2,5 +2,5 @@ package model
 
 // User represents a user.
 type User struct {
-	Name string
+	Name string `bson:"name"`
```

Comment:
Users don't really have just names for our purposes, so thoughts on renaming this field to `Username`?

Reply:
Only note on this, the field is currently name in production, so changing it would require a change there.

Reply:
I think that's fine. We should really have some automated migrations, but we don't, so I'll just make the change manually.